templates/header/topbar_end.html

```diff
@@ -67,6 +67,19 @@
                     <input id="nav-search" name="query" type="text" aria-label="Find crate by search query" tabindex="-1"
                         placeholder="Find crate" {%- if search_query %} value="{{ search_query }}" {%- endif %}>
                 </div>
+                {# The Search Sort by dropdown menu#}
+                <div id="search-select-nav"> 
+                    <label for="nav-sort">
+                        {{ "list" | fas }}
+                    </label>
+                    <select name="sort" id="nav-sort">
+                        <option value="revevance" {%- if search_sort_by and search_sort_by == "revevance" %} selected="selected" {%- endif %}>Relevance</option>
```

Comment:
```suggestion
                        <option value="relevance" {%- if search_sort_by and search_sort_by == "relevance" %} selected="selected" {%- endif %}>Relevance</option>
```

Reply:
It looks like a spelling mistake. I'll correct it later